app/components/emailSignup/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { LinkButton } from '../Button';
+import UserInput from '../common/userInput';
+import { Email } from '../icons';
+
+interface EmailSignupForm {
+  fullName: string;
+  email: string;
+  password: string;
+}
+
+const EmailSignup = () => {
+  const intialEmailFormState = {
+    fullName: '',
+    email: '',
+    password: '',
+  };
+  const [formState, setFormState] = useState<EmailSignupForm>(intialEmailFormState);
+
+  const updateFullName = (fullName: string) => {
+    setFormState((state) => ({ ...state, fullName }));
+  };
+
+  const updateEmail = (email: string) => {
+    setFormState((state) => ({ ...state, email }));
+  };
+
+  const updatePassword = (password: string) => {
+    setFormState((state) => ({ ...state, password }));
+  };
+
+  return (
+    <div data-testid="email-signup" className="flex flex-col w-full mt-4">
+      <UserInput
+        dataTestId="email-signup-name"
+        placeholder="Full Name"
+        label={''}
```

Comment:
Why are we not passing any value to the label here?

Reply:
> Why are we not passing any value to the label here?

I passed the value in the placeholder. Passing the same value in both places seems redundant. Let me know if the standard practice we follow is to mention it in label and not the placeholder or do we keep it in both places and I will change it accordingly.